### ISSUE TITLE
Output formats

### ISSUE THREAD

**Issue by ghost** [goshacmd/nsa_panel#3]
Please add additional output formats as HTML is hard to parse. See: www.prismstandard.org/faq

**Comment by jayvan** [comment]
Seconded, the hardware in [Room 641A](https://en.wikipedia.org/wiki/Room_641A) only supports RDF.

**Comment by ghost** [comment]
Word just came down... we need to provide the the general public (the Internet) unfettered access to our backdoor. Yeah. So can you make any security configurable so we can disable it? What about facebook/twitter auth just so we have some minimal level of logging/security and user profiles? How well does this thing scale btw? Also, we need to host our backdoor to the backdoor on usps.com... is that going to work for us? We'd still need output in a parseable format though, html's not going to work. Sorry... lot's of feature requests, I know, but the pointy-haired bosses are all stressed right now for some reason asking for changes to our stuff.

(note: this is all satire and a work of fiction and comedy... but we should have a progammatically accessible and open web, and this project would actually go a long way toward that... the problem is that it's only open and accessible to those with the master password... maybe I'll fork this... suggestions for a name? backdoor2frontdoor ?)

**Comment by ghost** [comment]
I know these issues were pretty big asks, so I forked the project. https://github.com/hollingsworthd/open_panel
